### DISCUSSION TITLE
When moving files, how to move to sub-directories?

### DISCUSSION THREAD

**Discussion by CountZeroOr** [comictagger/comictagger#681] [Q&A] [answered]
For the Renaming settings - there are options to move files when renaming. What I'd like to do is move files to directories based on Series Group and Series - how would I go about doing that?

**Comment by lordwelch** [comment, marked as answer]
Use `{series_group}/{series}`

**Comment by CountZeroOr** [comment]
That worked - I had the slash in the wrong direction.
